compiler/rustc_codegen_gcc/patches/libgccjit12/0001-core-Disable-portable-simd-test.patch

```diff
@@ -27,6 +27,6 @@ index d0a119c..76fdece 100644
  mod slice;
  mod str;
  mod str_lossy;
--- 
+--
 2.42.1
```

Comment:
hmm text editor settings, not sure if this breaks something

Reply:
Does it still work with the appropriate `git apply`?

Reply:
how would i test that? i have no idea how these are used.

Reply:
just standing in the repo and slapping in `git apply compiler/rustc_codegen_gcc/patches/libgccjit12/0001-core-Disable-portable-simd-test.patch`

checked and it does, this is gtg. :+1: